pyproject.toml

```diff
@@ -1,7 +1,61 @@
+[tool.poetry]
+name = "langchain"
+version = "0.0.26"
```

Comment:
i think we can remove the VERSION file in the langchain project now? also this should be bumped to 0.0.27

Reply:
Yeah, I was going to remove it in the next PR, but we can do it now.

https://github.com/shoelsch/langchain/commit/ed223dd3be7cd73454f2bff3abfde4f9aa1f2992